From 6521a4e7826b5d277d3fe6998c2bc537ec48d108 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Thu, 18 Dec 2014 11:59:19 -0500
Subject: [redhat] kabi: Adding symbol ioc4_unregister_submodule (drivers/misc/ioc4.c)

Message-id: <43f8e14d0fb3b341f32e2e61743c395c27fa31ed.1418903403.git.poros@redhat.com>
Patchwork-id: 102259
O-Subject: [PATCH RHEL7.1 27/48] kabi: Adding symbol ioc4_unregister_submodule (drivers/misc/ioc4.c)
Bugzilla: 1160760
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

Bugzilla: 1160760
    https://bugzilla.redhat.com/show_bug.cgi?id=1160760

    Description:
    ============
    The ioc4 is legacy hardware which we still need to support.
    These symbols are required to support some legacy oddball functionality.
    Used in ioc4_extint.ko

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/ioc4_unregister_submodule

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_x86_64/ioc4_unregister_submodule b/redhat/kabi/kabi-module/kabi_x86_64/ioc4_unregister_submodule
new file mode 100644
index 0000000..2a6b54f
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/ioc4_unregister_submodule
@@ -0,0 +1,2 @@
+#1-
+0xf0eb7163 ioc4_unregister_submodule drivers/misc/ioc4 EXPORT_SYMBOL
-- 
1.7.1